Library/ViewModels/RewardAddOnSelectionViewModel.swift

```diff
@@ -455,3 +460,36 @@ private func isValid(
   return latestSelectedQuantities != selectedRewardQuantities(in: backing)
     || backing.locationId != selectedShippingRule?.location.id
 }
+
+private func checkoutPropertiesData(project: Project, baseReward: Reward, rewards: [Reward]) -> KSRAnalytics
+  .CheckoutPropertiesData {
+  let staticUsdRate = Double(project.stats.staticUsdRate)
+  let estimatedDelivery = baseReward.estimatedDeliveryOn
+  let rewardMinimumUsd = Format.decimalCurrency(for: baseReward.minimum.multiplyingCurrency(staticUsdRate))
+  let rewardTitle = baseReward.title
+  let shippingEnabled = baseReward.shipping.enabled
+  let addOnsMinimumUsd = Format.decimalCurrency(
+    for: rewards
+      .reduce(0.0) { accum, addOn in accum.addingCurrency(addOn.minimum) }
+      .multiplyingCurrency(staticUsdRate)
+  )
+  return KSRAnalytics.CheckoutPropertiesData(
+    addOnsCountTotal: nil,
+    addOnsCountUnique: nil,
+    addOnsMinimumUsd: addOnsMinimumUsd,
+    amount: "",
```

Comment:
Is there a way we can avoid passing empty strings? Some of the other information available in this view model should make this possible.